### DISCUSSION TITLE
Feature Request: "Create Preset" enhancements

### DISCUSSION THREAD

**Discussion by Karl-Franz** [apohl79/audiogridder#1259] [Ideas]
These are just "quality of life" requests. Apologies if these have already been discussed or proposed:

- Cmd/Ctrl-N keyboard shortcut to Create a New preset
- Set cursor focus on Preset Name edit field when Create New Preset dialog opens so you can start typing name right away
- Remember last used preset folder when Create New Preset dialog opens rather going to root preset folder each time

**Comment by apohl79** [comment]
Thank you! I created a FR issue for this.
